baseplate/sidecars/event_publisher.py

```diff
@@ -92,6 +93,9 @@ def __init__(self, metrics_client: metrics.Client, cfg: Any):
         self.key_name = cfg.key.name
         self.key_secret = cfg.key.secret
         self.session = requests.Session()
+        self.session.headers.update(
```

Comment:
:no_bell: is there any advantage to doing this with `update()` over just `headers["User-Agent"] = "..."` ?

Reply:
Probably none at all :) Will change